0005-mod-history/README.md

```diff
@@ -0,0 +1,138 @@
+
+Both of these views show a time-sorted list of "subjects", each with a brief summary of the content ("Post on 2024-01-02 at 9pm", handle and avatar of an account, etc); timestamp of when the report or action took place (this is the sort order); and what the current status of the subject is (active, needs-review, suspended, takendown, deleted, etc).
+
+Clicking through on a subject is an additional view which shows:
+
+- click-through link to the content or account (if available)
+- summary of the content, but likely not the full content itself
+- current status of the content: takedown, content deleted, account deleted, account deactivated, etc
+- list of any labels applied by the current moderation service (labeler), with the full definition of the label (from declaration record)
+- limited history list of mod actions on the subject, each with a summary and timestamp
+- for account actions view, a button to appeal moderation action (eg, appeal label, appeal takedown)
+
+History would be relatively truncated/redacted. All reports and appeals by the current account would display. Any major public actions would be displayed (takedown, un-takedown, suspend, label). Ozone-private content would *definitely* not display (internal moderator comments, internal moderator tags), and the individual moderator account involved would *definitely* not be revealed. "Escalation" action would probably not display; "needs review" and "escalated" would probably look the same? In the future, it might be beneficial to indicate if an action taken was automated (eg, automod, auto-labeling).
+
+Email actions could be included for the recipient account ("Email sent at this time"). In the future, would be good to have the email body text display in-line, and/or create an additional event type which allows moderators to send arbitrary text messages/comments to users. However, it is not the intent for this interface/feature to become full bi-directional "chat" or "messaging".
+
+Metadata about subjects and reports should not be leaked. For example, an account should not be able to see if their content (or account) has been reported or not; that is private. Users should not be able to see if others have also reported a piece of content; that is private.
+
+In a future iteration, it might be helpful to retain previous versions of content ti display in the moderation history. For example, profile records are easily updated. If profile text or avatar image was updated in response to moderation action, it would be helpful for users to see the original record (which was actioned), and then any history of updates and the state of the record at that time, as part of the record. Similarly, self-deletion of content would be helpful to view as an event in history. Note that these events are *not* already stored in Ozone, and are *not* available via any existing service API.
```

Comment:
```suggestion
In a future iteration, it might be helpful to retain previous versions of content to display in the moderation history. For example, profile records are easily updated. If profile text or avatar image was updated in response to moderation action, it would be helpful for users to see the original record (which was actioned), and then any history of updates and the state of the record at that time, as part of the record. Similarly, self-deletion of content would be helpful to view as an event in history. Note that these events are *not* already stored in Ozone, and are *not* available via any existing service API.
```